integrations/gradient/pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "gradient-haystack"
 dynamic = ["version"]
 description = ''
 readme = "README.md"
-requires-python = ">=3.7"
+requires-python = ">=3.8"
```

Comment:
what's the reason for bumping?

Reply:
With `pip install gradientai` using a py37 environment, it installed `1.0.0.dev-something`